IBM/ibm-aix/ingest/parser.yml

```
@@ -86,5 +86,5 @@ stages:
   set_command_line_fields:
     actions:
       - set:
-          process.command_line: "{{parsed_event.message.process_name}} {{parsed_event.message.process_command_line}}"
-        filter: "{{parsed_event.message.process_name != None and parsed_event.message.process_command_line != None}}"
+          process.command_line: "{{parsed_event.message.process_name}} {{parsed_event.message.process_file_name}} {{parsed_event.message.process_command_line}}"
+        filter: "{{parsed_event.message.process_name != None and parsed_event.message.process_file_name != None and parsed_event.message.process_command_line != None}}"
```

As `process_file_name` is optional in the grok pattern, adding this condition will prevent having the command line for some event. I suggest duplicating these two lines, one example with the condition on process_file_name and the other without.

```suggestion
          process.command_line: "{{parsed_event.message.process_name}} {{parsed_event.message.process_command_line}}"
        filter: "{{parsed_event.message.process_name != None and parsed_event.message.process_command_line != None}}"
          process.command_line: "{{parsed_event.message.process_name}} {{parsed_event.message.process_file_name}} {{parsed_event.message.process_command_line}}"
        filter: "{{parsed_event.message.process_name != None and parsed_event.message.process_file_name != None and parsed_event.message.process_command_line != None}}"
```